wiki/Guides/Tablet_Purchase/en.md

```diff
@@ -48,22 +48,37 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
 
 ![](https://puu.sh/dlQH "Second generation Bamboo (Pen and Touch, Pen, Touch, Fun, and Craft - Bamboo Pen and Touch shown)")
 
-![](https://puu.sh/dlQi "Third generation Bamboo (Connect, Capture, Create - Connect shown)")
+![](https://wowcake.s-ul.eu/PgLyCagy "Third generation Bamboo (Connect, Capture, Create - Connect shown)")
```

Comment:
image should be included in PR. we don't support external hosting like this.

Reply:
Right, i'll change it to use the images that are already provided here.